src/components/Main/Main.scss

```diff
@@ -1,9 +1,10 @@
 $button-width: 100px;
 $button-spacing: 5px;
 
-.compare-button,
-.export-button,
-.run-button {
-  width: $button-width;
-  margin-left: $button-spacing;
+.controls button:not(:last-child) {
+  margin-right: $button-spacing;
+}
+
+.controls button {
+  min-width: $button-width;
```

Comment:
Allows buttons to grow a little for different languages to avoid wrap. See e.g. Deutsch.